back/engines/commercial/id_clave_unica/app/lib/id_clave_unica/clave_unica_omniauth.rb

```diff
@@ -83,6 +83,10 @@ def verification_prioritized?
       true
     end
 
+    def email_confirmed?(user)
+      !user.confirmation_required?
```

Comment:
I'm not sure why we need to confirm a user if confirmation is not required.

My idea was that this method checks the input data, not the user record. In `Base`, as you wrote, it's always `true`. In ClaveUnica, it'll always be `false`. In some other methods (e.g., in [Hoplr a bit later](https://github.com/CitizenLabDotCo/citizenlab/pull/6152/files#diff-7631f4d0fc7f405f7b5d2e3c3e9f834180b86488bf7e850722c4d4bdafa94536R59)), it will check the parameters from the SSO provider.

```suggestion
      false
```

Reply:
I see. I misunderstood.

What parameters from ClaveUnica should I check to decide if `authver_method.email_confirmed?(auth)`

Reply:
Just pushed the change. Feel free to question or change it.

> Which parameter(s) from ClaveUnica should I check to decide if authver_method.email_confirmed?(auth)

For now, we always know that ClaveUnica doesn't give us an email address, so we just always return `false`. But probably we'll use the `auth` object in other SSO providers (Hoplr) in the future.

Reply:
Makes sense :-)